### ISSUE TITLE
Triceps tendon rupture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10807]
**Preferred term label:**
Triceps tendon rupture

**Synonyms**
partial triceps tendon rupture or complete triceps tendon rupture

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Triceps tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001137
- label: Triceps tendon rupture
- parents: [Tendon rupture (HP:0100550)](https://hpo.jax.org/app/browse/term/HP:0100550)
- pmid: PMID:28832209
- def: Breakage (tear) of the triceps tendon.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10807
- synonyms:
